src/client/camera.h

```diff
@@ -176,10 +187,12 @@ class Camera
 	WieldMeshSceneNode *m_wieldnode = nullptr;
 
 	// draw control
-	MapDrawControl& m_draw_control;
+	MapDrawControl &m_draw_control;
 
 	Client *m_client;
 
+	CameraModeRestrictions &m_camera_mode_restrictions;
```

Comment:
remove the reference here please it's just an owned object on the stack

Reply:
Done